.github/workflows/haskell-ci.yml

```diff
@@ -12,19 +12,20 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        cabal: ["3.10.1.0"]
+        cabal: ["3.12.1.0"]
         ghc:
           - 9.0.2
           - 9.2.8
           - 9.4.8
-          - 9.6.5
-          - 9.8.2
+          - 9.6.6
+          - 9.8.4
+          - 9.10.1
```

Comment:
Do we really need to run 9.0.2 as well in CI now? If not, I'd say we should drop that.